### PR TITLE
Dockerfile for Irssi

### PR DESCRIPTION
**Pull request by trishnaguha**
Dockerfile for irssi

### CODE REVIEW

**Comment by sayanchowdhury** [comment]
Can you squash the commits?

**Comment by trishnaguha** [comment]
@sayanchowdhury Sure

File: irssi/README.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+```
+# docker build -t <username>/irssi .
+```
+
+To Run:
+-------
+
+```
+# docker run -it <username>/irssi
```
Comment (scollier) [inline irssi/README.md]: there are two docker runs here - they seem the same.  consolidate to one please? Also, do you really want to run it in interactive mode or detached?

File: irssi/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,11 @@
+FROM fedora
+MAINTAINER http://fedoraproject.org/wiki/Cloud
+
+# Updating dependencies, installing irssi and cleaning dnf caches to reduce container size
+RUN dnf -y update && dnf -y install irssi irssi screen && dnf clean all
+
+# Starting a new Screen session called chat
+CMD screen -S chat
```
Comment (scollier) [inline irssi/Dockerfile]: what is this CMD entry for?  I thought that only the last CMD command was respected.

File: irssi/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+
+# Starting irssi
+CMD irssi
```
Comment (scollier) [inline irssi/Dockerfile]: Please use the exec form here, with path.  Like: "CMD [ "/path/to/irssi" ]

File: irssi/README.md (hunk 1 above)
Comment (trishnaguha) [inline irssi/README.md]: @scollier I was thinking if we could add irssi to Fedora docker hub itself, then we could simply use `fedora/irssi` only.
`<username>/irssi` is for if people want to build themselves.
Yes we want to run it in interactive mode. Our purpose is to use irssi on atomic host. so more precisely this is for atomic users :).

Comment (scollier) [inline irssi/README.md]: OK.  Makes sense.  If you are targeting atomic users, should you leverage "atomic run" instead?

Comment (trishnaguha) [inline irssi/README.md]: Right. Thanks for your feedback @scollier

**Comment by trishnaguha** [comment]
@scollier Modified the PR with `atomic run` command.

**Comment by scollier** [comment]
@trishnaguha thanks for adding the atomic run option.

@baude any comments on this?

**Comment by scollier** [comment]
Thanks @trishnaguha LGTM.

**Pull request merged**